File: php/list_credentials.php
#!/usr/bin/env php
<?php
require '_client.php';
$trunk = getenv('TRUNK_SID');
if (!$trunk) {
    echo "Error: TRUNK_SID required\n";
    exit(1);
}
$q = [];
if (getenv('PAGE_SIZE')) {
    $q['page_size'] = getenv('PAGE_SIZE');
}
if (getenv('PAGE_OFFSET')) {
    $q['offset'] = getenv('PAGE_OFFSET');
}
if (getenv('CREDENTIAL_ID')) {
    $q['id'] = getenv('CREDENTIAL_ID');
}
echo "Listing credentials...\n";
exo_get("/trunks/$trunk/credentials", $q ?: null);

File: go/list_credentials.go
package main

import (
	"fmt"
	"os"
)

func main() {
	trunk := os.Getenv("TRUNK_SID")
	if trunk == "" {
		fmt.Println("Error: TRUNK_SID required")
		os.Exit(1)
	}
	q := map[string]string{}
	if v := os.Getenv("PAGE_SIZE"); v != "" {
		q["page_size"] = v
	}
	if v := os.Getenv("PAGE_OFFSET"); v != "" {
		q["offset"] = v
	}
	if v := os.Getenv("CREDENTIAL_ID"); v != "" {
		q["id"] = v
	}
	fmt.Println("Listing credentials...")
	get("/trunks/"+trunk+"/credentials", q)
}
